WORKSPACE

```diff
@@ -39,7 +39,7 @@ pip_deps()
 
 # Rust Toolchain + crates.io Dependencies
 load("//third_party/rust:repos.bzl", "rust_repos")
-rust_repos()
```

Comment:
I take it this is just a WIP for now, right?

Reply:
Indeed. I also have changes to rules_rust to support the riscv32 target that I'll need to upstream.